corehq/apps/app_manager/app_translations/validator.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import absolute_import
+
+from memoized import memoized
+from corehq.apps.app_manager.app_translations import (
+    expected_bulk_app_sheet_headers,
+    expected_bulk_app_sheet_rows,
+    get_unicode_dicts,
+)
+from corehq.apps.app_manager.app_translations.const import MODULES_AND_FORMS_SHEET_NAME
+
+COLUMNS_TO_COMPARE = {
+    'module_and_form': ['Type', 'sheet_name'],
+    'module': ['case_property', 'list_or_detail'],
+    'form': ['label'],
+}
+ROW_COUNT_MISMATCH_MESSAGE = "We found {} rows than expected. Validating what was uploaded."
```

Comment:
translate?

Reply:
@esoergel more/less is actually added later on depending upon the count.

> translate?
Looks like i cant add translation to it if i add more/less later. Going to break it down to two different message instead.

Reply:
Ah, I thought the `{}` was going to be replaced with a number.  Makes sense.

Reply:
There's also `We found {} rows but expected {}` as an alternative.